textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/SampleDataStore.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.dataflow.source;
+
+import static edu.uci.ics.textdb.dataflow.source.TestConstants.FIRST_NAME;
+import static edu.uci.ics.textdb.dataflow.source.TestConstants.LAST_NAME;
+import static edu.uci.ics.textdb.dataflow.source.TestConstants.SAMPLE_TUPLES;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field.Store;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.index.IndexableField;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+
+import edu.uci.ics.textdb.api.common.IField;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.common.StringField;
+import edu.uci.ics.textdb.dataflow.common.DataTuple;
+import edu.uci.ics.textdb.dataflow.constants.LuceneConstants;
+
+/**
+ * @author sandeepreddy602
+ *
+ */
+public class SampleDataStore {

```

Comment:
Will move SampleDataStore to textdb-store project and will make these changes in a different PR